tools/openshift/ocp-ipi-aws/ocp-ipi-aws-prep/ec2-resources.tf

```diff
@@ -23,6 +39,27 @@ resource "aws_security_group_rule" "worker_sg_vxlan_rule" {
   type                     = "ingress"
 }
 
+# Add a rule for vxlan traffic from master nodes to worker nodes.
```

Comment:
We also need to add master node to master node, like here:

https://github.com/submariner-io/submariner/blob/master/tools/openshift/ocp-ipi-aws/ocp-ipi-aws-prep/ec2-resources.tf#L17

Reply:
Oh, as talked in slack, in the existing IPI topology, the gateway node won't ever be part of the master SG.
and 4800 is used for node to GW communication.